sdk/typescript/test/e2e/txn-serializer.test.ts

```diff
@@ -177,7 +177,7 @@ describe('Transaction Serialization and deserialization', () => {
       gasBudget: DEFAULT_GAS_BUDGET,
       gasPayment: coins[0].objectId,
     } as MoveCallTransaction;
-    const serArgsExpected = await new CallArgSerializer(
+    const setArgsExpected = await new CallArgSerializer(
```

Comment:
```suggestion
    const serArgsExpected = await new CallArgSerializer(
```

This seems to be changed by the auto-formatter you are using